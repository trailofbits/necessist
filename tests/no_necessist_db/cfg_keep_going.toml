args = ["--root", "examples/cfg", "--keep-going"]
stderr = ""

[bin]
name = "necessist"

[status]
code = 0

[env.add]
TRYCMD="1"
