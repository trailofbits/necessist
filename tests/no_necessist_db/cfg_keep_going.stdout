4 candidates in 1 test file
examples/cfg/src/lib.rs: dry running
examples/cfg/src/lib.rs: mutilating
examples/cfg/src/lib.rs:5:5-5:12: Warning: Failed to run test `passed`

This may indicate a bug in Necessist. Consider opening an issue at: https://github.com/trailofbits/necessist/issues

examples/cfg/src/lib.rs:13:9-13:16: Warning: Failed to run test `timed_out`
examples/cfg/src/lib.rs:21:5-21:12: Warning: Failed to run test `failed`
examples/cfg/src/lib.rs:29:18-29:27: Warning: Failed to run test `nonbuildable`
